components/NavigationDrawer/src/private/MDCBottomDrawerContainerViewController.m

```diff
@@ -29,6 +28,7 @@
 // has the behavior as if we are scrolling at the end of the content, and the scrolling isn't
 // smooth.
 static const CGFloat kScrollViewBufferForPerformance = 20.f;
+static const CGFloat kFixedStatusBarHeightOnPreiPhoneXDevices = 20.f;
```

Comment:
Unfortunately this won't work as the height of the StatusBar can vary if there is an active call.
In the simulator under Hardware, use the Toggle In-call status bar.
We likely need to query the size of the status bar, and react to changes.